setup.py

```diff
@@ -66,96 +66,96 @@ def package_files(prefix: str, directory: str, extension: str):
     "torch>=1.9,<2",
     "yahp==0.1.0",
     "requests>=2.26.0,<3",
-    "numpy==1.21.5",  # TODO should this be >=1.21.5.10,<2
+    "numpy>=1.21.5.10,<2",
```

Comment:
Only update here

Reply:
```suggestion
    "numpy>=1.21.5,<2",
```
Removing the `.10` that is causing some jenkins errors